tests/flow_test.go

```diff
@@ -1402,6 +1402,57 @@ func TestPcapInject(t *testing.T) {
 	RunTest(t, test)
 }
 
+func TestDNSRetrieval(t *testing.T) {
+	test := &Test{
+		setupFunction: func(c *TestContext) error {
+			file, err := os.Open("../flow/pcaptraces/dns.pcap")
```

Comment:
There is already a test that use a pcap file with DNS packet. I would suggest to simply enhance it
https://github.com/skydive-project/skydive/blob/master/tests/flow_test.go#L1354